common/src/java/com/zimbra/common/soap/AdminConstants.java

```diff
@@ -1628,4 +1628,21 @@ public final class AdminConstants {
             "com_zimbra_docs_modern", "com_zimbra_drive_modern", "com_zextras_drive", "com_zextras_drive_open",
             "com_zextras_chat_open", "com_zextras_talk", "zimbra-zimlet-briefcase-edit-lool");
 
+    public static final String E_ACCOUNT_NAME = "accounts";
+
```

Comment:
Extra line spaces are not required here after every constant.

Reply:
done